src/parser.rs

```diff
@@ -349,7 +351,7 @@ impl<'a> Parser<'a> {
             '+' | '-' | '0'...'9' => {
                 while self.current.not_in(" \t\n\r#,]}") && self.inc() {}
 
-                // TODO EOF shittiness
+                // TODO: EOF shittiness

```

Comment:
Could you actually remove this comment and the block after it? The updated parser fixes this.